demos/logging-stack/logging_stack.h

```diff
@@ -34,10 +34,15 @@
 /* Standard Include. */
 #include <stdio.h>
 #include <stdint.h>
+#include <string.h>
+
+/* Macro to extract only the file name from file path to use for metadata in
+ * log messages. */
+#define __FILENAME__           ( strrchr( __FILE__, '/' ) ? strrchr( __FILE__, '/' ) + 1 : __FILE__ )
```

Comment:
I think we should avoid defining macros that begin with double underscore, since those can be reserved,

Reply:
Great point. Removed the double underscores